e2e.mk

```diff
@@ -14,9 +14,9 @@ test-e2e-run: generate-e2e-templates install-tools kind-create-bootstrap ## Run
     	-e2e.artifacts-folder="$(ARTIFACTS)" \
     	-e2e.config="$(E2E_CONF_FILE_ENVSUBST)" \
     	-e2e.skip-log-collection="$(SKIP_LOG_COLLECTION)" \
-    	-e2e.skip-resource-cleanup=$(SKIP_CLEANUP) -e2e.use-existing-cluster=$(SKIP_CREATE_MGMT_CLUSTER) $(E2E_ARGS)
-	$(MAKE) clean-release-git
+    	-e2e.skip-resource-cleanup=$(SKIP_CLEANUP) -e2e.use-existing-cluster=$(SKIP_CREATE_MGMT_CLUSTER) $(E2E_ARGS) \
```

Comment:
Do we need this trailing `\`?

Reply:
This one we don't need because `$AZWI_RESOURCE_GROUP` is the only environment variable context the cleanup script takes and we're already exporting that in the global make context.